CHANGES.rst

```diff
@@ -34,6 +34,9 @@ Internal Changes
   the ``can_manage`` method whose ``role`` argument has been renamed to
   ``permission`` (:issue:`3057`)
 
+- Add new ``registration_checkin_updated`` signal that can be used by
```

Comment:
there shouldn't be an empty line before this one (see the structore of the rest of the file)

Reply:
Should be fixed